paasta_tools/cli/cmds/spark_run.py

```diff
@@ -165,6 +184,13 @@ def get_spark_configuration(
     spark_conf['SPARK_EXECUTOR_CORES'] = args.executor_cores
     spark_conf['SPARK_EXECUTOR_MEMORY'] = '%dg' % args.executor_memory
 
+    if args.driver_max_result_size:
+        spark_conf['SPARK_DRIVER_MAX_RESULT_SIZE'] = '%dg' % args.driver_max_result_size
```

Comment:
Is the 'g' in here for GB? I yes, can we mention the unit in the help message?

Reply:
Yeah, the help message has it: 'Limit of total size in GB of serialized results of all partitions'